public/shiki/languages/abap.tmLanguage.json

```diff
@@ -0,0 +1,361 @@
+{
```

Comment:
we need to import these file assets because of this bug: https://github.com/shikijs/shiki/issues/138
but not all of them is loaded on server, just languages and theme requiredes